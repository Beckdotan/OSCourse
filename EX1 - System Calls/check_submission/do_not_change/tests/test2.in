X
2048 aux_files/from2.txt aux_files/to2.txt
