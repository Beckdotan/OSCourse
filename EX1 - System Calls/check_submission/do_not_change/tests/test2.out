Unable to open destination file for writing: No such file or directory
